recipes/spice-utilities/LICENSE.md

```diff
@@ -0,0 +1,78 @@
+
+### Acknowledgement
+Acknowledgement in your publications, tools and findings of use of SPICE or NAIF/PDS resources, and of the teams that provided the data used to make SPICE kernels, is encouraged; such community support will help ensure NASA will continue to fund the maintenance and further development of the SPICE system. Please check here for details.
+
+### Getting Help from NAIF
+NAIF is limited in the amount of help that may be offered with regard to SPICE. Prospective users of SPICE, including flight projects, proposers to NASA AOs, technology endeavors, and individual users, should carefully consider these limitations. Read the information on Support from NAIF available from this website.
+
+### Export
+SPICE software has been designated "Technology and Software Publicly Available" (TSPA) for export control purposes. As a result the export of SPICE software and related documentation is not restricted, and is independent of and not limited by export agreements for specific flight projects.
+
+However, anyone outside of the NAIF Group who produces a derived product incorporating SPICE software must obtain her/his own export designation.
+
+It is the finding of NASA's export control official that none of the SPICE system data, software, training or consultation are used for or useful in the design, fabrication, or operations of spacecraft systems or instruments, and thus publication or provision is not restricted under ITAR.
+
+To the extent that any SPICE information would take the form of software source code or non public technical data that is beyond the scope and nature of the scientific and performance information contained in the SPICE description provided to NASA, ITAR restrictions could apply.
+
+Publication or provision of project-specific SPICE data could be restricted under a flight project's own programmatic rules unrelated to ITAR regulations.
+
+### Clearance for Distribution
+JPL Document Review has issued Clearance CL#05-2438 for the distribution of SPICE products via the NAIF server.
+
+### Obtaining the SPICE Toolkit
+To meet NASA/JPL requirements and to better ensure getting the correct products, persons wishing a copy of the NAIF-supported SPICE Toolkit or any auxiliary utility programs should obtain it/them from this web site or from an official NASA flight project distribution. (Also see "Toolkit Redistribution" below.)
+
+### Toolkit Redistribution
+Redistribution of the unmodified SPICE Toolkit, such as from a mirror site, is prohibited without prior, written clearance from the NAIF manager. However, including the SPICE Toolkit modules (source and object code), documentation, and relevant SPICE Toolkit programs and allied User Guides as part of a package supporting a customer-built SPICE-based tool is entirely appropriate. This includes providing a new 3rd party interface to the SPICE Toolkit, subject to the relevant rules listed elsewhere on this webpage.
```

Comment:
This all sounds like we're not authorized to distribute this without written consent.

Reply:
Ah... I'll contact JPL and report back.

Reply:
Yes, they are OK with it. Email from [Boris Semenov, NAIF Manager](mailto:Boris.V.Semenov@jpl.nasa.gov):

```

-----Original Message-----
From: Boris Semenov <Boris.Semenov@jpl.nasa.gov> 
Sent: Tuesday, May 30, 2023 9:02 AM
To: Williams, Jacob
Cc: Semenov, Boris V (JPL-392N)[JPL Employee] <boris.v.semenov@jpl.nasa.gov>
Subject: Re: [EXTERNAL] Re: SPICE utilities in conda-forge

Jacob --

Thanks for the explanation. I am OK with you setting up a conda-forge recipe for the SPICE utility programs.

Regards,
Boris.
```